samples/javascript/E2_CopyFileToBlob/function.json

```diff
@@ -12,6 +12,5 @@
       "connection": "AzureWebJobsStorage",
```

Comment:
Other people might have different opinions, but I don't think that we should encourage people to use the same storage account as the function app (AzureWebJobsStorage). It's easier because it comes pre-provisioned but I think it's an easy way for users to shoot themselves in the foot (accidentally delete function execution-related objects in storage or do so many things in the account that they get throttled).

Reply:
I actually don't even really like this sample since it doesn't make a lot of sense for someone to do this anyways (backup your function content to blobs). We should consider creating a different sample for fan-out/fan-in - not for this PR, but at some later point.